apps/console/webpack.config.ts

```diff
@@ -36,6 +37,12 @@ import webpack, {
 import { BundleAnalyzerPlugin } from "webpack-bundle-analyzer";
 import DeploymentConfig from "./src/public/deployment.config.json";
 
+// Enable debug logs.
+const DEBUG_ENABLED: boolean = false;
+const DEBUG_LOG_KEY: string = "webpack-dev-server";
+// eslint-disable-next-line @typescript-eslint/typedef, no-console
+const log = console.log;
```

Comment:
If we directly assign `console`, we won't need a eslint disable comment here 😉 

```suggestion
const logger = console;
```

Reply:
PR is still drafted